src/profile-logic/process-profile.js

```diff
@@ -1460,11 +1460,12 @@ function _convertGeckoMarkerSchema(
     isStackBased,
   } = markerSchema;
 
-  const fields: GeckoDynamicFieldSchemaData[] = [];
```

Comment:
We still have `searchable` inside `GeckoDynamicFieldSchemaData`. I assume that was intentional since we haven't removed it from the gecko side yet. Could you also file a bug for this in bugzilla?